fsw/crypto_util/app/crypto_sequence.c

```diff
@@ -54,10 +57,11 @@ int main(int argc, char *argv[]) {
         debug_printf("File content: \n");
         debug_hexprintf(buffer,buffer_size_i);
 
+
         //Call Apply/ProcessSecurity on buffer contents depending on type.
         if (strcmp(security_type,"tc_a")==0){
-            Crypto_TC_ApplySecurity(&buffer, &buffer_size_i);
-        } else if (strcmp(security_type,"tm_a")==0){
+            Crypto_TC_ApplySecurity(buffer, buffer_size_i, &ptr_enc_frame, enc_frame_len);}
```

Comment:
enc_frame_len passed by value, instead of pointer to enc_frame_len (call should be identical to apply_security.c version)

Reply:
Also noticed this on 11/3 and communicated with CCutright. Fixed on the fly in the _cutright_dev_ branch

Reply:
Agreed - fixed in upcoming pull request.